hphp/runtime/vm/jit/vasm-arm.cpp

```diff
@@ -1545,7 +1533,7 @@ void lower(const VLS& e, movtdb& i, Vlabel b, size_t z) {
   lower_impl(e.unit, b, z, [&] (Vout& v) {
     auto d = v.makeReg();
     v << copy{i.s, d};
-    v << movtqb{d, i.d};
+    v << copy{d, i.d};
```

Comment:
But the register widths are propagated through copies, and they already don't match since this is a truncation operation...

Can you paste the exact error?